packages/eslint-plugin/tests/rules/no-useless-template-literals.test.ts

```diff
@@ -280,6 +280,52 @@ ruleTester.run('no-useless-template-literals', rule, {
       ],
     },
 
+    {
+      code: '`${Infinity}`;',
+      output: '`Infinity`;',
+      errors: [
+        {
+          messageId: 'noUselessTemplateLiteral',
+          line: 1,
+          column: 4,
+          endColumn: 12,
+        },
+      ],
+    },
+
+    {
+      code: noFormat`\`\${    Infinity    }\`;`,
+      output: '`Infinity`;',
+      errors: [
+        {
+          messageId: 'noUselessTemplateLiteral',
+        },
+      ],
+    },
```

Comment:
[Question] Is there a reason to add the `noFormat` tests for both of these? 
Like, is there any logic specific to formatting that's relevant for `Infinity` and/or `NaN`? I'd have thought it sufficiently covered by existing tests.

If not:

```suggestion
```

Reply:
Yeah, those are probably unnecessary. I just copy-pasted them from the previous one. removed :)